src/components/ArticleCard/ArticleCard.jsx

```diff
@@ -0,0 +1,20 @@
+import "./ArticleCard.css"
+
+function ArticleCard({ articleData }) {
+  const formattedDate = articleData.created_at.slice(0,10).split("-").reverse().join("-")
+  const capitalisedTopic = articleData.topic[0].toUpperCase() + articleData.topic.slice(1)
```

Comment:
Could be tidier to keep this string manipulation in reusable functions in a separate util file 🧹